### ISSUE TITLE
example for Fontawesome

### ISSUE THREAD

**Issue by Mullekular** [aslagle/reactive-table#203]
hi i try to get a fontawesome in the table. but i dont understand how. it lokks like this:

return {
                collection: Base.find({}, {sort: {createdAt: -1}}),
                rowsPerPage: 5,
                showFilter: true,
                useFontAwesome:true,
                showNavigation:true,
                showColumnToggles:true,
                fields: [
                    { key: 'start', label: 'Start' },
                    { key: 'goal', label: 'Ziel' },
                    { key: 'distance', label: 'Distanz' },
                    {key: 'font', <i class="fa fa-trash-o"></i>; },
            ]
            };

its my 2nd day in front end. im not very good in it. maybe you can show me an example of how i can become a fontawesome in the fields.

**Comment by Mullekular** [comment]
ok. i got ist :) i do it with a template.

**Comment by aslagle** [comment]
Yes, the useFontAwesome setting is only for icons in the table UI - the filter icon and the up/down arrows. You need custom templates to use it in fields :)
